src/peergos/shared/crypto/hash/PublicKeyHash.java

```diff
@@ -1,20 +1,22 @@
 package peergos.shared.crypto.hash;
 
+import jsinterop.annotations.JsType;
 import peergos.shared.cbor.*;
 import peergos.shared.io.ipfs.cid.*;
 import peergos.shared.io.ipfs.multihash.*;
 
+@JsType
 public class PublicKeyHash extends Multihash implements Cborable {
     public static final int MAX_KEY_HASH_SIZE = 1024;
     public static final PublicKeyHash NULL = new PublicKeyHash(new Multihash(Type.sha2_256, new byte[32]));
 
-    public final Multihash hash;
+    public final Multihash multihash;
```

Comment:
I wonder if we could get rid of this property all together?

Reply:
umm, not in this PR me thinks